src/Rating.soy

```diff
@@ -37,6 +37,7 @@
     <input
       type="hidden"
       aria-hidden="true"
+      name="{$inputHiddenName}"
```

Comment:
@mairatma. My goal about this input hidden is if in the case of this component be inside a web form. Do you really think is it a good idea? Or should we let the developer who needs passing this value through a web form implements this behaviour for himself?